### PR TITLE
F/first free

### PR DESCRIPTION
**Pull request by kduffie**

### CODE REVIEW

File: public/src/home/publicity-panel.html [hunk 1]
```diff
@@ -57,11 +57,9 @@
...
     <div class="content">
-      <h2>Early Publisher Bonus</h2>
-      <p class="row">We're offering daily cash bonuses to early content creators. For the first 48 hours after posting, in addition to what
-        you earn from viewers we will give you
-        <span class="price">[[newUserBonus]]</span> for each viewer. There is
-        <span class="price">[[remaining]]</span> remaining today.
+      <h2>Early Publisher Bonuses</h2>
+      <p class="row">As a card creator, in addition to what you earn from viewers, we are currently paying you
+        <span class="price">[[newUserBonus]]</span> for each viewer.
         <a href="https://channels.cc/c/cecb7191-9956-4a88-a45d-34c99247cdbf" target="_blank">Learn more.</a>
```
Comment (pshihn) [inline public/src/home/publicity-panel.html]: Minor personal preference: This sounds like it is like talking to existing publishers only. Previous wordings sounded like incentives to become a publisher. 
Don't care much either way though.

File: src/bank.ts [hunk 2]
```diff
@@ -402,6 +402,11 @@ export class Bank implements RestServer, Initializable {
     if (!networkInitiated && !user.admin && user.balance < details.amount) {
       throw new ErrorWithStatusCode(402, "Insufficient funds");
     }
+    if (forceAmountToZero) {
+      details.amount = 0.000001;
+    }
+    const totalAmount = forceAmountToZero ? 0.000001 : details.amount;
+
```
Comment (pshihn) [inline src/bank.ts]: Redundant check, no?
const totalAmount = details.amount;
will do the same.

File: src/network-entity.ts [hunk 3]
```diff
@@ -131,6 +132,7 @@ export class NetworkEntity implements Initializable {
     const totalCoins = this.publisherSubsidyMinCoins + Math.round(Math.random() * (this.publisherSubsidyMaxCoins - this.publisherSubsidyMinCoins));
     if (!subsidyDay || midnightToday > subsidyDay.starting) {
       console.log("Network.poll: Adding new publisher subsidy day", totalCoins, this.publisherSubsidyCoinsPerOpen);
+      const amount = await cardManager.calculateCurrentPublisherSubsidiesPerPaidOpen(subsidyDay ? subsidyDay.coinsPerPaidOpen : 0.50);
       await db.insertPublisherSubsidyDays(midnightToday, totalCoins, this.publisherSubsidyCoinsPerOpen, this.publisherSubsidyReturnUserBonus);
```
Comment (pshihn) [inline src/network-entity.ts]: Not sure if there's a bug here, but you are not using the 'amount' value anywhere.

File: public/src/home/publicity-panel.html (hunk 1 above)
Comment (kduffie) [reply public/src/home/publicity-panel.html]: Okay.  I've rewritten it.  See if you like the revised version any better.  (Or suggest another version.)

File: src/bank.ts (hunk 2 above)
Comment (kduffie) [reply src/bank.ts]: Yes.  Side-effect of multiple passes of editing.

File: src/network-entity.ts (hunk 3 above)
Comment (kduffie) [reply src/network-entity.ts]: That was a bug alright.  the "publisherSubsidyCoinsPerOpen" has been replaced with a dynamically calculated "amount" but wasn't using it to populate the right field.  Good catch.

File: public/src/home/publicity-panel.html (hunk 1 above)
Comment (pshihn) [reply public/src/home/publicity-panel.html]: It's fine. I am not sure what I really want. Good to go for now.

**Pull request merged**